test/common/README.md

```diff
@@ -50,6 +50,12 @@ Platform normalizes the `dd` command
 
 Check if there is more than 1gb of total memory.
 
+### envPlus(additionalEnv)
+* return [&lt;Object>]
+
+Returns `process.env` plus `additionalEnv`. Used to pass a temporarily modified
```

Comment:
I think the word `temporarily` is misguiding here. The environment of the child process is modified for its entire lifetime, and the enviroment of the calling process is left unchanged.